### PR TITLE
Change backend from generic weather API to topics

### PR DESCRIPTION
**Pull request by RenderMichael**
Here is my proposal for the initial topics implementation for the backend. What do you think, @David-Render?

### CODE REVIEW

File: src/BackEnd/TopForum.DotNetBackend/TopForum.API/Program.cs [hunk 1]
```diff
@@ -16,29 +16,45 @@
 
 app.UseHttpsRedirection();
 
-var summaries = new[]
+List<Topic> topics =
```
Comment (David-Render) [inline src/BackEnd/TopForum.DotNetBackend/TopForum.API/Program.cs:19]: Please add more topics!! :)

**Review by David-Render — CHANGES_REQUESTED** [review]
Overall good progress!

File: src/BackEnd/TopForum.DotNetBackend/TopForum.API/Program.cs (hunk 1 above)
Comment (RenderMichael) [reply src/BackEnd/TopForum.DotNetBackend/TopForum.API/Program.cs:19]: OK I'm on it

```txt
David is a programmer. He is gonna be a pro.
```

Comment (RenderMichael) [reply src/BackEnd/TopForum.DotNetBackend/TopForum.API/Program.cs:19]: All done, PTAL.

**Review by David-Render — APPROVED** [review]
Good topics! We will work on more.

**Pull request merged**